bridge/autoload.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+if (getenv('DD_AUTOLOAD_NO_COMPILE') === 'true') {
+    // Development
+    $files = include __DIR__ . '/_files.php';
+    foreach ($files as $file) {
+        require_once $file;
+    }
+} else {
+    // Production
+    require_once __DIR__ . '/_generated.php';
```

Comment:
I am retracting my 👍. I agree that this would be a bug, but why we should have a bug to cause a crash when we can have that same bug not to cause a crash? Instead what I am proposing is to change the [`require_once`](https://github.com/DataDog/dd-trace-php/pull/840/files#diff-5e02ea6e1ce4e3bd4dfb13879ad38660R7) on line 7 to `require` that would still cause a crash if called twice but in development. WDYT?

Reply:
@morrisonlevi pinging you on this comment as GH removed from the files view as the file has changed because of the push to change line 7.

Reply:
If it changes to require for "development", is there a chance that users hit it? Users will still use the PSR-4 autoloader, so I think they won't. If so then it's good enough for me.